src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -512,13 +512,20 @@ def wait_for_write_image_to_disk(self, nodes_count: int = MINIMUM_NODES_TO_WAIT)
             nodes_count=nodes_count,
         )
 
-    def wait_for_host_status(self, statuses, fall_on_error_status=True, nodes_count: int = MINIMUM_NODES_TO_WAIT):
+    def wait_for_host_status(
+        self,
+        statuses,
+        fall_on_error_status=True,
+        nodes_count: int = MINIMUM_NODES_TO_WAIT,
+        fall_on_pending_status=False,
```

Comment:
@eliorerz "fall" was an intended wording or is it a typo for "fail"?

Reply:
its fall_ in all places  ... because i guess pending is not a failure

Reply:
Yeah, it's a wierd name to me, the goal is to raise en error when the boolean is set https://github.com/openshift/assisted-test-infra/blob/e8af7bad44d692ff65818e40eaaa4408fbefdeaa/src/assisted_test_infra/test_infra/utils/waiting.py#L33-L40

Anyway, I approve as it is already made like that.

Reply:
ack Thanks !